plugins/brightness/PageComponent.qml

```diff
@@ -79,12 +79,11 @@ ItemPage {
         Column {
             anchors.left: parent.left
             anchors.right: parent.right
-
-            ListItem.Standard {
+            
+            //SettingsItemTitle to be consistent with the rest of settings
+            SettingsItemTitle {
```

Comment:
If this is consistent with the rest of the codebase, there is no need for a comment here.

Reply:
done, also realized I needed to remover the dividers for the titles as all the others do the same